libs/core/src/schemas/commands/contest.schemas.ts

```diff
@@ -0,0 +1,48 @@
+import z from 'zod';
+import { ContestManager } from '../entities.schemas';
+
+export const CreateContestManagerMetadata = {
+  input: z.object({
+    contest_address: z.string(),
+    name: z.string(),
+    image_url: z.string(),
+    funding_token_address: z.string(),
+    prize_percentage: z.number(),
+    payout_structure: z.array(z.number()),
+    interval: z.number().min(0),
```

Comment:
on the front end I have only availability to have `yes` / `no` answers for the question if the contest is recurring. How does it map to the numbers? Is it:
0 - non recurring
1 - recurring

Reply:
I believe so– @Rotorsoft can you confirm?

Reply:
according to [this line](https://github.com/hicommonwealth/commonwealth/pull/7580/files#diff-38efe3656fe6d8518f7a30e2db118d778a665e43cea0d6272519a7b9db9e48e7R28), on protocol side it should be number of milliseconds. As far as I know, the only recurrence we want right now is one week. @ianrowan @Rotorsoft can we clarify this?

Reply:
This should actually be seconds